administrator/components/com_media/tmpl/file/default.php

```diff
@@ -72,4 +75,4 @@
     <?php endif; ?>
     <input type="hidden" name="mediatypes" value="<?php echo $this->escape($mediaTypes); ?>">
 </form>
-<script type="module" src="<?php echo $script . '?' . $this->getDocument()->getMediaVersion(); ?>"></script>
+<script type="module" src="<?php echo $script . '?' . $this->getDocument()->getMediaVersion(); ?>"></script>
```

Comment:
```suggestion
<script type="module" src="<?php echo $script . '?' . $this->getDocument()->getMediaVersion(); ?>"></script>

```
Please add back the line feed at the end of file.